index.html

```diff
@@ -10,9 +10,19 @@
     <!-- Title -->
     <title>YOSHITHA RATHNAYAKE PRO</title>
     <link rel="icon" type="image/png" href="images/YOSHITHA RATHNAYAKE PRO.png" />
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <h1>YOSHITHA RATHNAYAKE PRO</h1>
-    <p>This is The Official Web Site Of YOSHITHA RATHNAYAKE PRO</p>
+
+    <nav>
```

Comment:
Remove the nav tag bcs this isn't the nav I told to add it as an img @avezqureshi14